app/slimming/slimming.js

```diff
@@ -6,6 +6,29 @@ app.controller('GOSlimCtrl', function($scope, $location, $q,
   $scope.selection = {};
   $scope.deSelectedItems = [];
 
+
+
+  function sticky_relocate() {
```

Comment:
Why not reuse the same strategy and code used to make the nav bar float on the term page? See `term.js`